vllm/config.py

```diff
@@ -299,6 +300,30 @@ def _verify_quantization(self) -> None:
                     f"method specified in the `quantization` argument "
                     f"({self.quantization}).")
 
+        # TODO(alpin): Tune the default exp_bits for each fp_bits
+        DEFAULT_EXP_BITS = {
+            4: 2,
+            5: 2,
+            6: 2,
+            7: 3,
+        }
+        quant_llm_methods = [
+            "fp4_weights", "fp5_weights", "fp6_weights", "fp7_weights"
+        ]
```

Comment:
Could you keep these in and import from the quant_llm.py file to have one source of truth?